src/game-of-life/deploying-to-production.md

```diff
@@ -1,25 +1,44 @@
-# Production and Deployment
+# Deploying our Game of Life Web Application to Production
 
 When we're happy with the project, the next step is to deploy it to a production
-server instead of our development server.
+server instead of our local development server. This is generally the same with
+Rust and WebAssembly as it is with anything else: put the files somewhere where
+they are exposed to the Web via an HTTP server!
 
-The first step is to run:
+We make sure our wasm module is an up-to-date build by running `wasm-pack` from
+within the `wasm-game-of-life` directory:
 
 ```
-npm run bundle
+wasm-pack init
```

Comment:
This creates a `pkg` directory though?

Reply:
It makes the things below a bit unclear as to what is happening.

Are we making a `pkg` directory and `webpack` knows to put this in `www`? Overall I'm confused as to what's going on here

Reply:
Because of `npm link` any changes to the `pkg` directory are reflected in `www`'s linked dependency on `wasm-game-of-life`.